crates/loam-cli/src/commands/init.rs

```diff
@@ -64,10 +67,39 @@ impl Cmd {
         copy_example_contracts(&self.project_path)?;
         rename_cargo_toml_remove(&self.project_path, "core")?;
         rename_cargo_toml_remove(&self.project_path, "status_message")?;
+        update_workspace_cargo_toml(&self.project_path.join("Cargo.toml"))?;
         Ok(())
     }
 }
 
+// update a soroban project to a loam project
+fn update_workspace_cargo_toml(cargo_path: &Path) -> Result<(), Error> {
+    let cargo_toml_str = read_to_string(&cargo_path).map_err(|e| {
+        eprintln!("Error reading Cargo.toml file in: {cargo_path:?}");
+        e
+    })?;
+
+    let cargo_toml_str = regex::Regex::new(r#"soroban-sdk = "[^\"]+""#)
+        .unwrap()
+        .replace_all(
+            cargo_toml_str.as_str(),
+            r#"loam-sdk = "0.6.12"
+loam-subcontract-core = "0.7.5""#,
```

Comment:
Todo:
- [ ] clean up this ugly string
- [ ] fetch the most recent versions of these crates instead of hardcoding them

Reply:
Is there an easy way to grab it from the cloned frontend repository? That will already have the latest dependencies, and it might be surprising if it mismatches what's generated by `loam init`.

Reply:
If you're going to do this raw string approach, I think the outdented thing is totes fine. But maybe I'm just used to seeing it; we do it a lot in the guts of the TS Bindings code, and in the `build_clients` tests in this repo.